Task_7/DeliveryTrucks/truck_delivery_rayus007/driver_manager.py

```diff
@@ -0,0 +1,27 @@
+import redis
+import json
+from truck_delivery_rayus007.util.content_utils import UID
+from truck_delivery_rayus007.content_manager import ContentManager
+
+
+class DriverManager(ContentManager):
+    """ Class representing a DriverManager """
+    def __init__(self):
+        super().__init__()
+        self.cache = redis.Redis(host='localhost', port=6379, db=0)
+
+    def save_document(self, document):
+        self.cache.set(f"{UID}", json.dumps(document))
+
+    def get_all(self):
+        list_result = []
+        for key in self.cache.scan_iter(f"{UID}_*"):
```

Comment:
nice, this would be moved to the dbconnnector class to about doing this in every manager

Reply:
It's now in the db_connector_redis.py (in the next task)